### PR TITLE
Feature/863 in kind contribution to candidate (take 2)

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
#863 
#864

[API PR](https://github.com/fecgov/fecfile-web-api/pull/490)
[Validator PR](https://github.com/fecgov/fecfile-validate/pull/252)

### CODE REVIEW

**Comment by Elaine-Krauss-TCG** [comment]
Initial testing hasn't revealed any issues, but this needs further testing and more code coverage before I'll be finished with it...

File: front-end/src/app/shared/components/inputs/committee-input/committee-input.component.ts [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
   @Input() entityRole = 'CONTACT';
   @Input() includeFecId = false;
   @Input() readonly = false;
-
-  ngOnInit(): void {
-    this.form
-      .get(this.templateMap.organization_name)
-      ?.valueChanges.pipe(takeUntil(this.destroy$))
-      .subscribe((value) => {
-        this.form.get(this.templateMap.committee_name)?.setValue(value);
```
Comment (toddlees) [inline front-end/src/app/shared/components/inputs/committee-input/committee-input.component.ts:20]: Is there a place in the new code where the committee_name is "bound" to the organization_name when `organization_name` and `committee_name` need to be redundant?

If not, the transacion type form util could add a subscription on the condition that the transactionType has the flag, copyOrgNameIntoCommitteeName.  that value could default to true in the transactionType class, and be overwritten to false in the transactions you've now introduced.

**Review by toddlees — DISMISSED** [review]

File: front-end/src/app/shared/components/inputs/committee-input/committee-input.component.ts (hunk 1 above)
Comment (Elaine-Krauss-TCG) [reply front-end/src/app/shared/components/inputs/committee-input/committee-input.component.ts:20]: Matt wound up re-adding this, so this issue should be resolved

**Comment by Elaine-Krauss-TCG** [comment]
image

**Review by mjtravers — APPROVED** [review]

**Pull request merged**